operator/tests/common/central-cr.yaml

```diff
@@ -14,6 +14,14 @@ spec:
         cpu: 1
     adminPasswordSecret:
       name: admin-pass
+    db:
```

Comment:
I wonder whether this will still fit in a default-sized `infra` cluster. :thinking: